src/python/py/models/builder.py

```diff
@@ -2602,6 +2769,7 @@ def get_args():
                 enable_cuda_graph = 1 : The model can use CUDA graph capture for CUDA execution provider. If enabled, all nodes being placed on the CUDA EP
                     is the prerequisite for the CUDA graph to be used correctly. It is not guaranteed that cuda graph be enabled as it depends on the model
                     and the graph structure.
+                use_8bits_moe = 1 : Use 8-bit quantization for MoE layers. Default is using 4-bit quantization.
```

Comment:
Can you add an example for `use_8bits_moe` in `README.md` [after the CUDA graph example](https://github.com/microsoft/onnxruntime-genai/tree/main/src/python/py/models#enable-cuda-graph)?.

Reply:
sure